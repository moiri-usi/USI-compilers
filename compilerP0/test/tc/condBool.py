a = 6
b = 7
c = 42
print 1, a == 6
print 2, a == 7
print 3, not a == 6
print 4, not a == 7
print 5, a == 6 and b == 7
print 6, a == 7 and b == 7
print 7, not a == 7 and b == 7
print 8, a == 7 or b == 7
print 9, a == 7 or b == 6
print 10, not (a == 7 and b == 6)
print 11, not a == 7 and b == 6
print 12, a != 6
print 13, a != 7
print 14, a < 7
print 15, a > 5
print 16, a <= 6
print 17, a >= 6
print 18, a > 6
print 19, b < 7
