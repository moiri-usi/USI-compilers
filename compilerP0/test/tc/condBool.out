1 True
2 False
3 False
4 True
5 True
6 False
7 True
8 True
9 False
10 True
11 False
12 False
13 True
14 True
15 True
16 True
17 True
18 False
19 False
